applications/HDF5Application/python_scripts/file_utilities.py

```diff
@@ -0,0 +1,64 @@
+import KratosMultiphysics as KM
```

Comment:
this file is a collection of functions that I imagine can be useful also for other things in this app